src/common/components/ReviewItem/ReviewItem.tsx

```diff
@@ -1,25 +1,23 @@
 import { reviewItemTheme, type ReviewItemVariants } from "./ReviewItem.theme";
-import { ReviewBody } from "./ReviewBody";
-import { ProfileSchool } from "./ProfileSchool";
-import { ProfileReviewer } from "./ProfileReviewer";
-import { getHumanReadableTimestampDelta } from "@/common/functions";
-import { ThumbUpFilledIcon } from "@/common/components/CustomIcon";
 import { LockCtaOverlay } from "@/common/components/LockCtaOverlay";
-import { Button } from "@/common/components/Button";
-
+import ReviewHeader from "@/common/components/ReviewItem/ReviewHeader";
+import ReviewModal from "@/common/components/ReviewModal/ReviewModal";

```

Comment:
chore(non-blocking): should use relative import for imports on the same level - storybook will sometimes complain about recursive relative import and break (see https://github.com/AfterClass-io/afterclass.io-v2/pull/98)

Reply:
https://github.com/AfterClass-io/afterclass.io-v2/pull/104/commits/79fb3d79570fd8a8455ef4e70dcc687af5bbf8b4